extension_templates/transformer.py

```diff
@@ -241,3 +235,42 @@ def _inverse_transform(self, X, y=None):
         # IMPORTANT: avoid side effects to X, y
         #
         # type conventions are exactly those in _transform, reversed
+
+    # todo: consider implementing this, optional
+    # if not implementing, delete the method
+    def get_fitted_params(self):
+        """Get fitted parameters.
+
+        Returns
+        -------
+        fitted_params : dict
+        """
+        # implement here
+
+    # todo: return default parameters, so that a test instance can be created
+    @classmethod
```

Comment:
Make this static rather than class you don't use the cls instance in any of the examlpes i've seen